src/iris/ui/static/js/iris.js

```diff
@@ -1972,6 +1994,20 @@ iris = {
         iris.createAlert('Error: Failed to load reprioritization data -' + response.text);
       });
     },
+    getCustomNotificationSettings: function() {
+      var self = this;
+      return $.getJSON(this.data.url + this.data.user + '/categories').done(function(response){
+        self.data.settings.categories = response;
+        var tempSet = new Set();
```

Comment:
Instead of using `tempSet` as a variable name, why not something more descriptive like `applicationNames`?